### DISCUSSION TITLE
slideshow drag event

### DISCUSSION THREAD

**Discussion by ceccaHF** [uikit/uikit#4547] [Q&A] [answered]
Hi,
is it possible to detect the drag event on the slideshow and which direction?
I need to show a progressbar related to the active slide. Everything works on click, swipe, scroll but not on drag.
Thank you !

**Comment by ceccaHF** [comment, marked as answer]
found out a solution with the 'itemshow' event. This works with any type of interaction:

```
var previous_slide = 0;
  $slideItem.on('itemshown', function(){
    var slide_count = $('.uk-slideshow-items li').length;

    var current_slide = UIkit.getComponent(document.querySelector('[uk-slideshow]'), 'slideshow').index;
    previous_slide = current_slide;

    animateProgressbar(slide_count, current_slide);

  });
```

**Comment by vitali-duckma** [comment]
use swiper js
